packages/playout-gateway/src/tsrHandlerJobs/job.ts

```diff
@@ -0,0 +1,18 @@
+interface IJob<OwnResultType, PrevResultType> {
```

Comment:
Hmmm something about avoid `I...` interface naming, as it is a standard not used in typescript.

I for one do not mind it, but i know Lindved has mentioned it a bunch of times when i was developing the TricasterIpInfo and also to Ida, as she also have a tendency to add it. You could say it is a work damage, from being mainly a C# dev.